### ISSUE TITLE
basic communication needs

### ISSUE THREAD

**Issue by boxuancui** [HealthAPI/helmet#25]
- IM group: what do you use?
- mailing list: what is your personal Email? What about Glenn?
- anything else you can think of?

**Comment by gtallen1187** [comment]
my personal email is gtallen1187@gmail.com.  glen's is glen.fernandes2@gmail.com.

In terms of real-time communication, i typically use google hangouts.  i know glen uses this too.  however, i typically use this just to chat among friends, so if we're looking for a more robust communication platform for collaborative software development, we should use [HipChat](https://www.hipchat.com/).  i've used it in the past and found it to be incredibly useful.  also, more importantly - it's free.

there is a mobile app for both iOS and android too, so if we want to use it as our primary communication platform, there's no requirement that it needs to be up-and-running on your desktop in order to get messages.  It's a great platform, and i enjoy using it.  works great with group conversations too, and keeps a history so that you can trace back through conversations you may have missed and read what was discussed.  plus, it integrates directly with GitHub - meaning that we can have github post to the chatroom whenever commits are made or pull requests are merged, etc.  

However, if all we need is a simple way to ping one another, hipchat might be overthinking it a bit.  honestly, i'm open to anything here.  my only worry is that we may potentially discourage future contributors if we demand that they learn too many new platforms.

then again, those might be exactly the type of people we _want_ to discourage...

Basically, i'm open to anything.  i use google hangouts all day, i know glen does too, but hipchat is definitely more powerful.  not sure what dawid prefers.

**Comment by gtallen1187** [comment]
also @boxuancui, especially once you leave, it might be worth thinking about setting up an in-person meeting every 3-4 weeks or so.  it could be for an hour after work one day, or a couple hours on a saturday, but i think that it will definitely help to meet in person every now and then going forward, especially during the architecture/design phases of the project.

**Comment by boxuancui** [comment]
I agree. Let me set up the mailing list first, so that calendars can be marked. I will think about Google Hangout or HipChat.

**Comment by boxuancui** [comment]
@HealthAPI/owners, updates and thoughts on this issue:
1. I am waiting for Matt's Email and we are good to go for the mailing list. Once finalized, I will send an Email to everyone from there.
2. [Google Hangouts](http://www.google.com/+/learnmore/hangouts/) or [HipChat](https://www.hipchat.com/). I would vote for latter, because it has good integration with GitHub. Imagine we want to talk about specific commits, HipChat will be a good tool to reference. Otherwise, we can use it as a regular chat client.

Ideas? Objections?

**Comment by glfernandes88** [comment]
-I would go with HipChat since the UI facilitates 'chatting' within specific threads or groups so it might be scalable moving forward. ( I am sure Hangout does the same, but it'll interfere with my work's Hangout so I give it a thumbs down :p )

-Also an option similar to Groups could be Basecamp which me, Matt & Greg have used extensively to maintain to-do's, product development stuff etc in the past 

Also, my # is 716 390 2746, if any of you guys are in the South End over the weekend, feel free to give me a call & we can touch base

**Comment by gtallen1187** [comment]
i'm gonna say HipChat as well - the chat room interface is perfect for what we want to do i think, and since none of our companies use it, seems like it wouldn't interfere with much.  plus as @boxuancui said, the integration with github is great - automatically posts commits and pull request right to the chat room, and since you can look back through all chat records, there's really no chance of missing anything.

plus, @matthewplunkett was telling me the other day that HipChat has recently really stepped up their emoji game :grinning: :stuck_out_tongue_winking_eye: :sunglasses: 

And @glfernandes88 - since its clear from your `:p` attempt above that your emoji powers have gotten rusty, here is the [official emoji reference guide](http://www.emoji-cheat-sheet.com/)

**Comment by matthewplunkett** [comment]
@HealthAPI/owners sorry for the delay guys, just getting caught up with this. i'm a fan of HipChat and use it everyday, but would also be perfectly content with Hangouts. just to complicate things, a few weeks back i read [Slack](https://slack.com/) is converting people and superior to HipChat. Free to use unless you need more than 5 integrations.

email: mattplunkett16@gmail.com
phone: 857-939-9135

**Comment by boxuancui** [comment]
@HealthAPI/owners:
- HipChat home page: https://healthapi.hipchat.com/home
- Group mailing list: healthapi@googlegroups.com
- Google Groups home page: https://groups.google.com/d/forum/healthapi

I will leave this open in case there are additional questions and concerns. If nothing is updated, I will close this issue in 2 weeks.
